megamek/docs/history.txt

```diff
@@ -1,6 +1,7 @@
 VERSION HISTORY:
 ----------------
 v0.43.4-git
++ PR #635: Terrain Master SPA (All but Drag Racer and Nightwalker)
```

Comment:
We generally append new entries onto the bottom.  The top most entry is the first thing that got fixed/added in a release, and subsequent entries happen after that.

Also, I think it's better to enumerate the SPAs added rather than list the ones that weren't (it's more informative to people who don't know all of the terrain master SPAs.  You could just say "Terrain Master SPAs: blah, blah 2, and blah 3".

Reply:
So for future reference or go ahead and change it?

Also, the only spot it is called from is where I changed it in Hex.java Can have it changed in 5-10 if you still think it may be an issue in the future 👍

Reply:
This isn't really written down anywhere yet, but I think the policy we should generally have is that the person merging in the PR should add the history.txt entries.  The reason is, any subsequent changes to history.txt are all but guaranteed to cause a merge conflict.  This means we'll have to sort out this conflict for every PR.  In this case, I asked you to add them because then I can merge this PR before I get home tonight, and I don't expect anyone to update history.txt before then.

Yes, I think it's best to ensure that `Terrain.movementCost` never returns negative.  I think it's a good preventative measure.